.github/workflows/build.yml

```diff
@@ -1,17 +1,17 @@
 name: build
 
 on:
-  - push
-  - pull_request
+    - push
+    - pull_request
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version: 16
-          cache: yarn
-      - run: yarn install
-      - run: yarn build
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+            - uses: actions/setup-node@v3
+              with:
+                  node-version: 18
```

Comment:
Current LTS is 20; 18 is just in maintenance mode: https://endoflife.date/nodejs